compiler/rustc_traits/src/implied_outlives_bounds.rs

```diff
@@ -70,8 +70,9 @@ fn compute_implied_outlives_bounds<'tcx>(
         // FIXME(@lcnr): It's not really "always fine", having fewer implied
         // bounds can be backward incompatible, e.g. #101951 was caused by
         // us not dealing with inference vars in `TypeOutlives` predicates.
-        let obligations = wf::obligations(infcx, param_env, hir::CRATE_HIR_ID, 0, arg, DUMMY_SP)
-            .unwrap_or_default();
+        let obligations =
+            wf::obligations(infcx, param_env, hir::def_id::CRATE_DEF_ID, 0, arg, DUMMY_SP)
```

Comment:
Could you import `CRATE_DEF_ID` directly too everywhere useful?

Reply:
yeah, I also prefer using an import, even in that case